webapp/src/components/datablocks/CycleBlock.vue

```diff
@@ -278,7 +278,7 @@ export default {
 }
 
 .limited-width {
-  max-width: 650px;
+  max-width: 100%;
```

Comment:
Can we also port this limited width to the normal bokeh block? I'd like to then play around with different widths a bit (and see if we can even have something resizable...)